### DISCUSSION TITLE
Select dropdown not working on Safari if the form has a password field

### DISCUSSION THREAD

**Discussion by Sartoric** [filamentphp/filament#13240] [Help]
### Package

Form builder

### Package Version

3.2.83

### How can we help you?

I have a form with a password TextInput

```
                TextInput::make('password')
                    ->password()

```
and a multiple Select with a custom relationship

```
                Select::make('group')
                    ->multiple()
                    ->relationship('roles', 'name', modifyQueryUsing: fn (Builder $query) => $query->where('system', true))

```

The dropdown works ok when selecting the first value, but if a try to select a second one, it suddenly closes when the value is found. Clicking on it again I can select the value found.

The problem is that using preload() method for Select, the dropdown, after one value has already been selected, immediately closes making it useless

Removing the password field (even just the method) seems to fix the issue.

It seems to be Safari related (using v17.5 19618.2.12.11.6).
Chrome (125.0.6422.142) seems to works fine

Anyone faced the same issue ? 

I've tried to file a bug report but since a repo is needed and atm I have no time to build a demo project  (and the filament-issue template seems to be basically an empty laravel project without filament installed)  I guess it will be auto-closed

**Comment by yslinear** [comment]
facing the same issue!
